src/components/picture/_children/source.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import formatSrcset from "../../../utils/format-image-resizer-src";
+
+const Source = ({ height, media, resizedOptions, resizerURL, src, width, ...rest }) => {
+	if (!media || !src) return null;
```

Comment:
Shouldn't this be an `&&` as both props are marked with `isRequired`

```suggestion
	if (!media && !src) return null;
```

Reply:
If neither of them are there, it will render null, which is appropriate.

`If not media or not src`, render null if either of them are missing.

`If not media and not src` would mean it renders if one of them is there.

Reply:
The alternative is to wrap the whole return with `if (media && src)` which is the boolean inversion of the original.

Reply:
I updated to use the De Morgan boolean inversion.